seasonConfigScript.js

```diff
@@ -20,7 +20,7 @@ function addNewSeason()
     var isAlreadyStored = false;
     if (document.cookie.length > 0) //if cookies are stored at all
     {
-        if (getCookie("seasonList") !== "")
+        if (getCookie("seasonList") !== " ")
```

Comment:
`getCookie("seasonList").strip().length > 0`
would be more consistent, doesn't care about whitespace or not.